java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/dataaccess/api/ApplicationPersistenceEntity.java

```diff
@@ -24,6 +24,9 @@
   /** @see #getModificationCounter() */
   private int modificationCounter;
 
+  /** @see #getRevision() */
+  private Number revision;
```

Comment:
why did we add an unused field here? This seems pointless. Am I getting the diff in the wrong directions? Removing an unused field would make sense to improve SonarQube issues but vice versa this is confusing me.